.gitignore

```diff
@@ -9,7 +9,7 @@ src/bitcoin-gui
 src/bitcoin-node
 src/bitcoin-tx
 src/bitcoin-wallet
-src/test/fuzz
+src/test/fuzz/*
 !src/test/fuzz/*.*
```

Comment:
Aren't these two contradicting?

Reply:
The above ignores all files in the folder, but not the whole folder
The below un-ignores any file with an extension (e.g. `.h` or `.cpp` files)

Effectively, files with no extension are ignored (e.g. binary executable files), no?